scheduler/cloudformation/cloudformation.go

```diff
@@ -721,6 +740,8 @@ func (s *Scheduler) Instances(ctx context.Context, app string) ([]*scheduler.Ins
 			return instances, err
 		}
 
+		ec2InstanceID := aws.StringValue(containerInstances[*t.ContainerInstanceArn].Ec2InstanceId)
```

Comment:
Awesome. Didn't know about about `aws.StringValue`. We can replace the `safeString` method with calls to this instead (doesn't need to be in this PR).